releasenotes/notes/add-aix-support-0170d23332663a08.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM - add AIX support for the trace-agent
```

```suggestion
    APM: Add support for AIX.
```
We generally use the `APM: ` prefix, and the `trace-agent` mention is not needed since it's implied by APM.